_posts/2019-03-12-google-imagery-deprecation.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Note from the Director Regarding the Google Imagery Deprecation'
+author:
+  display_name: Matt Peters
+  email: mpeters@utah.gov
+date: 2019-03-12 16:03:11
+categories:
+  - Featured
+tags:
+  - Google Imagery
+  - deprecation
+  - Discover services
+  - 
+---
+The State of Utah has enjoyed the relationship we have had with Google related to the purchase of the Google Imagery product. Our original contract with Google began in January 2015, so we have been grateful to have collaborated with Google this long. Moving forward, however, we will be looking for a new imagery source, as Google will no longer offer the imagery program for states, of which Utah was an early adopter. AGRC will be working with other states, as well as AppGeo, the vendor that serves [the Imagery]({{ "/discover/" | prepend: site.baseurl }}), to come up with other imagery sources. There is a possibility that Google may make a 2019 flight available for purchase, but as it is not certain, we must move forward securing another source regardless of that possibility.
+
+Thank you all for your support, both as users of the Google Imagery product and as financial supporters of this valuable resource. I have already submitted funding requests to some organizations, so to those organizations: thanks for your consideration of those requests. Please know that those requests are still important and valid as we pay for the 2018 imagery and look for a new source. We appreciate your continued support as we work to encourage and facilitate the effective use of geospatial information and technology for Utah. 
+ 
+ 
+[Matt Peters](mpeters@utah.gov)  
```

Comment:
I do not like to have emails and phone numbers in documents as they change and are difficult to maintain. That is why we maintain the _data/contacts.yml file - to centralize this information.  

```suggestion
[site.data.contacts.agrc_director.name](site.data.contacts.agrc_director.email)  
```